### ISSUE TITLE
cell growth mode switching, bipolar to monopolar TPV?

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#15463]
I know we have discussed this many times before, but I can't work out if something has changed 
and it is now incorrect, or it was always like this and  I  misremebered

I want to use the term
https://www.ebi.ac.uk/QuickGO/term/GO:0051524
GO:0051524    cell growth mode switching, bipolar to monopolar

(I think this is the term we use for NETO, although it does not have these synonymns.)

I'm also pretty sure that in the past this did not have a parentage to growth (because it isn't to do with  a size increase, its to do with the direction of growth)

Anyway, it now has a parent to "growth" via regulation of cell growth.

@ukemi 
@mah11

**Comment by mah11** [comment]
First, you simply must mean GO:0051523, which does have narrow synonyms for NETO. GO:0051524 is switching in the other direction.

Second, I can't see how these could possibly _not_ be regulation of growth. Neither switch regulates the extent of growth, but both regulate where cell growth happens. Regulation of cell growth itself (GO:0001558) explicitly includes regulation of growth direction as well as extent and rate. The current structure looks correct.

I also don't think it's changed recently - some background in #12645 and #8713.

**Comment by ValWood** [comment]
> First, you simply must mean GO:0051523
yes I do

so in GO growth is defined as:
The increase in size or mass of an entire organism, a part of an organism or a cell.


This has nothing to do with an increase in size (it isn't regulating the size increase at all, the growth will happen, but not in the direction usually specified).

So to me, annotating these  to growth just seems really wrong? this is morphology (shape), not size?

 
Personally I don't think it should include direction. Direction isn't about size. Otherwise we  have no way to  distinguish things which are involved in a size increase?

This always confuses me but I though I understood it at one point....

**Comment by ValWood** [comment]
I still don't get this. I just read the old ticket and I  still think something is odd.

How can this be correct?

GO:0016049    cell growth
Definition (GO:0016049 GONUTS page)
The process in which a cell irreversibly increases in size over time by accretion and biosynthetic production of matter similar to that already present.

regulation of cell growth:
GO:0001558   
Any process that modulates the frequency, rate, extent or direction of cell growth.


Surely the regulation definition needs to be the   same as the parent term?
We can't just add direction onto the def can we?

**Comment by ValWood** [comment]
"regulation of cell size" has no relationship to cell growth?
https://www.ebi.ac.uk/QuickGO/term/GO:0008361

**Comment by mah11** [comment]
I remain satisfied that the reasoning documented in the older tickets is sound, and the current ontology structure in this area is correct.

**Comment by ValWood** [comment]
but the definition of cell growth would need to be updated in this case? to reflect that it includes direction, not only size. It is currently logically inconsistent.

**Comment by hdrabkin** [comment]
but since directional growth is still growth, I would think there is no inconsistency.....

**Comment by ValWood** [comment]
> but since directional growth is still growth, I would think there is no inconsistency.....

No, I still think this is a key issue.
Direction of growth governs cell shape.
Cell shape and cell size are *completely separable*.

We have 2 choices. I don't know which one is correct.

1. We either  extend the definition of cell growth to include  "directional growth". To me this seems wrong.

or 
2.
We separate "growth" from "direction of growth.

I will try to illustrate with 2 examples.

In S. pombe cell morphology mutants  the cell size (volume) is  not altered.  Fission yeast is usually cylindrical,  but is   directional growth cues are lost, cells can be pear shaped, rounded or spherical (depending on the severity of the phenotype). In these, growth is not affected. The cell size is ~ the same.

In a second example,  normal cells grow from one end (monopolar growth), until New End Take-off (= NETO=bipolar growth). In mutants where NETO   is compromized,  cells grow from only one end. However in these mutants cell size ids NOT affected. Cell size is the same.

These  processes (direction of growth) seem to be more to do with shape (morphology) than size (growth).  This seems to be an issue of semantics. If we define growth as an increase in size (as we currently have, and I should continue, but I could be convinced otherwise), then "direction of growth/growth polarity", despite having "growth" in it's term name is not really "growth" at all. 

The reason I think this is that the 2 processes are completely separable (they are controlled by different pathways), although clearly there is  cross talk.

It really is very important because most gene products involved  in cell polarity (shape), have absolutely nothing to do with cell  growth or size). They are independent  (although clearly there is linking between these at some level through  CDK activation).

I don't think I explained that very well ;).

I can ask some experts who work specifically on growth and polarity  who are trying to figure out the connections between the two. It's a pretty active area of research.....

**Comment by mah11** [comment]
Direction of growth is an attribute of the growth process; it is not separable from growth. Controlling growth direction is therefore a form of regulating growth.

In other words, growth is adding matter. The new matter has to go somewhere. Normal growing cells can control how much new mass they add (regulation of extent of growth), how fast it's added (regulation of growth rate), and where new mass goes (regulation of growth direction).

The text definition of "cell growth" doesn't need to have direction added explicitly, as it correctly says that cell growth is adding matter to get bigger. But it would not be actively incorrect to add a gloss saying that any aspect of the growth process - extent, rate, direction - can be regulated.

The pombe examples show that there should not be any is_a or part_of links between the types of growth _regulation_ (reg of extent, rate, direction are and should remain siblings under regulation of cell growth), but they are all consistent with the logical and biological reality that each type of regulation, well, regulates some aspect of cell growth.

On a tangent: cell morphology can be controlled either by regulating growth direction or without growth (by rearranging existing matter). Yeasts don't really do the latter, but lots of other species do.

tl;dr: I oppose any changes to the current ontology structure in this area.

**Comment by ukemi** [comment]
I agree with @mah11. Changing shape without growth is not growth, but directional growth is still growth.

**Comment by mah11** [comment]
For me, the key is that growth direction can be regulated independently of regulating growth extent, but both are still ways to regulate growth.

**Comment by ValWood** [comment]
Finally, I get it (again). I think a def gloss would be very, very  useful here (even if I'm the only person on the planet who needs it- it will save you time in the future).

Not to self:
https://www.ebi.ac.uk/QuickGO/term/GO:0061387
regulation of extent of cell growth
is the term I need for "cell growth, size increase"

Phew, that took me a while!

**Comment by ValWood** [comment]
no issue.
